opentreemap/treemap/lib/user.py

```diff
@@ -98,6 +109,11 @@ def get_audits(logged_in_user, instance, query_vars, user, models,
     if exclude_pending:
         audits = audits.exclude(requires_auth=True, ref__isnull=True)
 
+    # Uncomment to dump the large, complex query for debugging
```

Comment:
I am going to just remove it. It is easy enough to write this in by hand in the rare cases when you want to dig through a big SQL statement.